textalyze.js

```diff
@@ -14,15 +14,28 @@ function itemCounts(array) {
   return counts;
 }
 
-module.exports = { itemCounts };
+/**
+ * Given an input String, returns an Array containing each letter
+ * @param {String} string - The String to convert to array
+ * @param {Array} array - The Array containing the characters from the string
+ */
+function arrayFrom(string) {
+  if (!string || typeof string !== 'string') {
```

Comment:
Hmm, tell me more about why you think the best thing to do here is return `[]` if we're not given a string?  The following return values would surprise me:

```javascript
Object.is(arrayFrom(12345), arrayFrom('12345')) === false
Object.is(arrayFrom((12345).toString()), arrayFrom('12345')) === true
Object.is(arrayFrom(12345), arrayFrom('')) === true
Object.is(arrayFrom('12345'), arrayFrom('')) === false
```

Reply:
I thought it was better to avoid errors, since the input will be a text from somewhere, but 
after reading your comparison, the behavior is a little strange. I guess I should turn a number into an array too